modules/hal_nordic/nrfx/nrfx_kconfig.h

```diff
@@ -490,8 +487,6 @@
 #define NRFX_SPIM4_ENABLED 1
 #endif
 
-#define NRFX_SPIM_DT_HAS_RX_DELAY(node) DT_PROP(node, rx_delay_supported) +
-
 #if DT_FOREACH_STATUS_OKAY(nordic_nrf_spim, NRFX_SPIM_DT_HAS_RX_DELAY) 0
```

Comment:
`NRFX_SPIM_DT_HAS_RX_DELAY()` is tightly coupled with this line, so I think it would be better to keep it here.

Reply:
Moved